README.md

```diff
@@ -4,12 +4,13 @@ Kawaii Gherkin
 [![Build Status](https://travis-ci.org/malukenho/kawaii-gherkin.svg?branch=master)](https://travis-ci.org/malukenho/kawaii-gherkin)
 [![Code Coverage](https://scrutinizer-ci.com/g/malukenho/kawaii-gherkin/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/malukenho/kawaii-gherkin/?branch=master)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/malukenho/kawaii-gherkin/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/malukenho/kawaii-gherkin/?branch=master)
+[![Code Climate](https://codeclimate.com/github/malukenho/kawaii-gherkin/badges/gpa.svg)](https://codeclimate.com/github/malukenho/kawaii-gherkin)
```

Comment:
Seems unnecessary, given that scrutinizer-ci is already covering the quality with more detail